README.md

```diff
@@ -16,3 +16,29 @@ chmod +x hook_setup.sh
 in order to set up git hooks
 
 ssh-working hell yeahhhh!!
+
+## Database:
+
+To correctly setup a database, you need to run a replica set and be connected to that. This can be done locally for development purposes using the run-rs npm package. To start it (given that mongodb service 4.0 or above is installed) run the following command (remember the datafolder is always relative the path you run from).
+```
+run-rs --keep --mongod --dbpath <path/to/data/folder> &
+```
+start the mongo-shell and make sure you use the database 'cloudnews' using 
```

Comment:
One important thing is to be on the primary node, or does these commands work on secondary?

Reply:
k

Reply:
Done